### DISCUSSION TITLE
DataTable Frozen Row alignment options

### DISCUSSION THREAD

**Discussion by florian-gebele-spit** [primefaces/community#2113] [PrimeVue]
Hi!

At the moment we are only able to freeze a row and make it stick to the top. I propose a feature, similar to freezing columns where we are able to align it to left and right, just for rows where we have the option to stick it to the bottom or top.
Is anything similar planned already?

**Comment by vieira-brz** [comment]
Hello!

As of now, many spreadsheet and data grid applications typically include the functionality to freeze rows at the top or columns on the left. The ability to freeze rows at the bottom or align columns to the right is less common and not standard in most popular tools like Excel, Google Sheets, or even in many data grid libraries.

If you're asking about a specific software or library (like Excel, Google Sheets, or a JavaScript data grid library), there is no widely known plan to implement such a feature. However, this could be a valuable addition for many users, especially for presenting totals or summary data that needs to remain visible as you scroll through other data.

For custom implementations or if you're developing an application:
- **Custom Software Development**: You might consider implementing this feature by adjusting the CSS for display positioning in web applications or using similar properties in other types of software.
- **Contribution to Open Source**: If you are using an open-source library, you could propose this feature to the maintainers or contribute the code yourself.

If this feature is crucial for your work, checking the documentation or forums of the specific tool you are using or reaching out to their support or community might also provide information on any upcoming features or workarounds.
